tests/parser/types/numbers/test_sqrt.py

```diff
@@ -0,0 +1,63 @@
+from decimal import (
+    ROUND_FLOOR,
+    Decimal,
+)
+
+
+def decimal_sqrt(val):
+    return val.sqrt().quantize(
+        Decimal('0.0000000000'),
+        rounding=ROUND_FLOOR
+    )
+
+
+def test_sqrt_literal(get_contract_with_gas_estimation):
+    code = """
+@public
+def test() -> decimal:
+    return sqrt(2.0)
+    """
+    c = get_contract_with_gas_estimation(code)
+    assert c.test() == decimal_sqrt(Decimal('2'))
+
+
+def test_sqrt_variable(get_contract_with_gas_estimation):
+    code = """
+@public
+def test(a: decimal) -> decimal:
+    return sqrt(a)
+
+@public
+def test2() -> decimal:
+    a: decimal = 44.001
+    return sqrt(a)
+    """
+
+    c = get_contract_with_gas_estimation(code)
+    val = Decimal('33.33')
+
+    assert c.test(val) == decimal_sqrt(val)
```

Comment:
This doesn't seem to work for values less than 1.
```python
        c = get_contract_with_gas_estimation(code)
        val = Decimal('0.999') # Decimal('33.33')
        print(c.test(val))
    
>       assert c.test(val) == decimal_sqrt(val)
E       AssertionError: assert Decimal('0.9995') == Decimal('0.9994998749')
E        +  where Decimal('0.9995') = <function gas_estimation_decorator.<locals>.decorator at 0x7f7f91e8e2f0>(Decimal('0.999'))
E        +    where <function gas_estimation_decorator.<locals>.decorator at 0x7f7f91e8e2f0> = <tests.conftest.VyperContract object at 0x7f7f91e629e8>.test
E        +  and   Decimal('0.9994998749') = decimal_sqrt(Decimal('0.999'))

tests/parser/types/numbers/test_sqrt.py:40: AssertionError
```

Reply:
found the bug:

`if z >= y:` instead of `if z == y:`.

and will write a test accordingly ;)